### ISSUE TITLE
Release on October 28, 2016

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#7397]
The scheduled release on Friday, October 28, 2016 is assigned to:

@kaicataldo
@ilyavolodin 

Please use this issue to document how the release went, any problems during the release, and anything the team might want to know about the release process. This issue should be closed after all patch releases have been completed (or there was no patch release needed).

**Comment by kaicataldo** [comment]
We had an issue with the deploy (missing a `replacedBy` key for the deprecated rule) that caused site generation to fail. @ilyavolodin was able to generate the site locally, but we weren't able to generate the changelog/release notes, so we did that manually.

Post-mortem: Can we add guard so that `replacedBy` isn't necessary? Also, could we maybe make the release notes generation a separate task (or at least the change log) so that we can run it locally should this happen again in the future?

@eslint/eslint-team Problems have been resolved, and the release was successful. Please do not merge anything other than documentation and bug fix PRs until we determine on Monday whether we need a follow up patch release or not.

**Comment by kaicataldo** [comment]
@eslint/eslint-team Patch release was successful - please continue merging all PRs as usual. Thanks!
